packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -15,8 +15,7 @@ export enum BaseQueryTypes {
 
 export enum CacheTags {
     engineStatus = 'engineStatus',
-    // This is a placeholder for now. Having only one messes up the type of the tag in the onCacheEntryAdded function.
-    testStatus = 'testStatus',
+    subscription = 'subscription',
```

Comment:
Doesn't subscriptionDetail better describe the cache value?

Reply:
I think it should be subscription. My thought: when setting the cachekey in `subscriptionDetail.ts` the subscriptionId is set as ID:
```
providesTags: (result, error, arg) => [
                {
                    type: CacheTags.subscription,
                    id: arg.subscriptionId,
                },
            ],
```
Invalidating this key would mean you will invalidate a specific subscription with id: yy